features/atcryptoauth/sample_ssl_client.cpp

```diff
@@ -0,0 +1,393 @@
+/* mbed Microcontroller Library
```

Comment:
This is a port of ```ssl_client.c``` and is very specific to ATCAECC508A i.e. ECDSA and SHA256 cipher suite. 

It may be removed after this [PR](https://github.com/ARMmbed/mbed-os-example-tls/pull/148). But as that example gets more general I would like to preserve as a demonstration of ATACECC508A device.